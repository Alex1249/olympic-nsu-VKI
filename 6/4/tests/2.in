12
115
115
116
116
117
118
119
119
121
121
122
123
30
116
112
115
116
112
115
105
114
118
111
111
116
116
110
108
113
112
117
118
112
114
114
112
115
110
116
120
115
112
107
