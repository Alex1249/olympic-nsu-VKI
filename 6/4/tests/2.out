3
1
3
7
7
10
1
1
7
5
3
7
11
9
4
4
3
-1
4
3
4
5
4
7
4
-1
7
10
10
4
